scripts/create_draft_release.sh

```diff
@@ -10,6 +10,11 @@ set -o pipefail # prevents errors in a pipeline from being masked
 
 RELEASE_TAG=$1
 
+if ! [[ "$RELEASE_TAG" =~ ^[0-9]\.[0-9]\.[0-9](-.+)?$ ]]; then
```

Comment:
I think we won't be able to release any major, minor or patch version that is equal to or greater than 10. 
I think at some point this will hit us.

Reply:
Ah your right, let me adjust that

Reply:
```suggestion
if ! [[ "$RELEASE_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
```